CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.3.6
+
+* Extend `run_test` to use `CountryTaxBenefiSystem` _or_ `country_tax_benefit_system` from country openfisca package
```

Comment:
I don't understand the purpose of this change: in which situation is `country_tax_benefit_system` necessary?

Can you add something in the changelog line? :)